StreamChatClient_v3/APIClient/Endpoints/Payloads/ChannelCreatePayload.swift

```diff
@@ -0,0 +1,58 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+public struct ChannelCreatePayload<ExtraData: ExtraDataTypes>: Encodable {
+    let cid: ChannelId
+    let name: String?
+    let imageURL: URL?
```

Comment:
`name` and `imageURL` are already part of the extra data, already, so they shouldn't be here.